### ISSUE TITLE
kmc_tools complex does not accept  non-alphanumeric characters in out_db_path  

### ISSUE THREAD

**Issue by fplaza** [refresh-bio/KMC#11]
Hi,
Here are some examples that should work but produce errors

 operations_definition_file_1 (with dot)
```
INPUT:
sample0 = sample0.31mers 
sample1 = sample1.31mers 
OUTPUT:
samples_union.31mers=sample0+sample1
```
```
kmc_tools complex
Error: wrong line format, line: 5
```

operations_definition_file_2 (with slash)
```
INPUT:
sample0 = sample0.31mers 
sample1 = sample1.31mers 
OUTPUT:
samples_union/samples_union=sample0+sample1
```
```
kmc_tools complex
Error: wrong line format, line: 5
```

**Comment by marekkokot** [comment]
Hi,
Thank you. It was fixed in 706282963537a701c4b9fedd5e722f2989a0b469 
Those changes are not inlcuded in current binaries available on the website yet (it will be with the next release). For now I would suggest compile current sources from github. If you have compiled newest version from github and problem remains please let me know here.

**Comment by fplaza** [comment]
I don't have a C++14 compatible compiler to test the github version.
However, it seems to work fine now http://cpp.sh/9fgqe
Thanks.
